### PR TITLE
Use new sidebar api to register groupfolders section

### PR DESCRIPTION
**Pull request by juliushaertl**
Requires https://github.com/nextcloud/server/pull/16312

### CODE REVIEW

File: src/files.js [hunk 1]
```diff
@@ -30,30 +33,15 @@
...
-	OC.Plugins.register('OCA.Sharing.ShareTabView', ShareTabPlugin);
 })(OC, OCA);
+
+window.addEventListener('DOMContentLoaded', () => {
+	import(/*c webpackChunkName: "sharing" */'./SharingSidebarApp').then((Module) => {
+		OCA.Sharing.ShareTabSections.registerSection((el, fileInfo) => {
+			if (fileInfo.mountType !== 'group') {
+				return
+			}
+			return Module.default
```
Comment (skjnldsv) [inline src/files.js]: interesting, dynamic import needs us to specify `defaukt` ?

Comment (juliushaertl) [reply src/files.js]: yes, it will return an object with all the exports

**Comment by skjnldsv** [comment]
Shall we merge?

**Comment by juliushaertl** [comment]
Yes, we need to make sure to have a version release just for 18 then

**Pull request merged**